src/components/mainContent/DirectionButtons.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./mainContent.css";
+import arrows from "../../assets/upArrow.svg";
+import content from "../../assets/labContent.js";
+export default function DirectionButtons(props) {
```

Comment:
Overall, I will say this implementation is a little messy, in the sense that it may be hard to work with in the future. However, if you feel like you know what's going on generally, we can merge it for now.

Reply:
It works for now, but please tell us how we can clean it up so we can fix it up in the future!